udata/core/site/metrics.py

```diff
@@ -140,7 +140,7 @@ class MaxReuseDatasetsMetric(SiteMetric):
     def get_value(self):
         reuse = (Reuse.objects(metrics__datasets__gt=0).visible()
                  .order_by('-metrics.datasets').first())
-        return reuse.metrics.get('datasets', 0)
+        return reuse.metrics['datasets'] if reuse else 0
```

Comment:
Not needed as `objects(metrics__datasets__gt=0)` ensure the `datasets` metrics is present on the reuse.